hipchat.go

```diff
@@ -168,10 +183,20 @@ func (c *Client) RoomHistory(id, date, tz string) ([]Message, error) {
 	uri := fmt.Sprintf("%s/rooms/history?auth_token=%s&room_id=%s&date=%s&timezone=%s",
 		c.BaseURL, url.QueryEscape(c.AuthToken), url.QueryEscape(id), url.QueryEscape(date), url.QueryEscape(tz))
 
-	resp, err := http.Get(uri)
+	reqs, err := http.NewRequest("GET", uri, nil)
```

Comment:
drop the trailing s from reqs, so that it’s `req`.

Here and elsewhere you’ve made the change.